practices/managing-static-assets.md

```diff
@@ -0,0 +1,13 @@
+# Managing static assets
+
+Static assets are files that don't need to be generated, modified or processed by an application before being delivered to the user. This typically includes CSS, JavaScript, images and web fonts. Static assets change rarely, and their content doesn't depend on user input or preferences.
+
+We host these kinds of static resources on our own infrastructure and Content Delivery Network (CDN) instead of including them on our pages from a third-party CDN. This applies especially to JavaScript files or libraries, and font files.
+
+Self-hosting static assets has several advantages over relying on third parties for their delivery:
+
+* **Resilience.** As long as our site is up, the static resource will be up too. We don't depend on third party servers or additional CDNs besides our own, and we won't be affected by their outages, or closure of the service.
```

Comment:
```suggestion
* **Control.** As long as our site is up, the static resource will be up too. We don't depend on third party servers or additional CDNs besides our own, and we won't be affected by their outages, or closure of the service.
```
This is not to do with _resiliency_. We are simply swapping one point of potential failure (someone' else's server) for another one (our own server). This is instead to do with _control_.

A _resilient_ solution would be (for instance), falling back to our own servers were the 3rd party ones down.